packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.base.tsx

```diff
@@ -69,6 +69,10 @@ export class CommandBarBase extends BaseComponent<ICommandBarProps, {}> implemen
 
     this._classNames = getClassNames(styles!, { theme: theme! });
 
+    // ResizeGroup will render these attributes to the root <div>.
+    // TODO We may need to elevate classNames from <FocusZone> into <ResizeGroup> ?
+    const nativeProps = getNativeProps<React.HTMLAttributes<HTMLDivElement>>(this.props, divProperties, ['data', 'className', 'styles']);
```

Comment:
divProperties should already filter out data and styles, right?

Reply:
There IS `style` in `divProperties`.
https://github.com/OfficeDev/office-ui-fabric-react/blob/93d5e98a4b3859ce196b53dcda230edbb5ad0478/packages/utilities/src/properties.ts#L107

I will remove `data` here, but there is still exclusion rule of `data` in `ResizeGroup.base.tsx`.
https://github.com/OfficeDev/office-ui-fabric-react/blob/3432f59416afac5a6f8535bb35bdba8bf438439f/packages/office-ui-fabric-react/src/components/ResizeGroup/ResizeGroup.base.tsx#L329